src/robusta/cli/main.py

```diff
@@ -202,7 +202,7 @@ def gen_config(
             else:  # self registration
                 account_name = typer.prompt("Choose your account name")
                 email = typer.prompt(
-                    "Enter a GMail/GSuite address. This will be used to login"
+                    "Enter a Gmail/Google Workspace address. This will be used to login"
```

Comment:
Two brand items -- note that `GSuite` was rebranded...

Reply:
> Oh, gosh. we _use_ MS Teams (and Slack). I never managed to parse `msteams` as `ms teams` -- I always parsed it as `m-steams` and wondered "Is this m-streams?"

Haha, yep :)